recipes/klib/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "klib" %}
+{% set version = "0.0.69" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 17611d0416af42d034d3532c624fde45677af45a0adc8edb83dd1556979961df
+
+build:
+  number: 0
```

Comment:
This package looks like it can be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I did include noarch: python initially, which resulted in a failed build. I've included it again and specified >=py3.6 in the host and run requirements, which apparently works!